src/test/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupportSpec.scala

```diff
@@ -325,16 +446,10 @@ class DataUseRestrictionSupportSpec extends FreeSpec with Matchers with DataUseR
         }
         "should handle populationRestrictions string lists" in {
           val attrs = generateStructuredUseRestrictionAttribute(new DuosDataUse(
-            populationRestrictions = Some(Seq(
-              "population 1",
-              "population 2"
-            ))
+            pediatric = Some(true)
```

Comment:
This looks like a very different test now. Why?

Reply:
There was a (potential) bug that was translating RS-POP to RS-PD codes. @ahaessly is this truly a bug? If so, should I change the test to check pediatric codes (as I did here) or simply delete the test?

Reply:
Confirmed that it was a real bug.

Reply:
Wow. Good catch!